src/mscorlib/shared/System/TimeZoneInfo.Unix.cs

```diff
@@ -395,6 +397,87 @@ private static string FindTimeZoneIdUsingReadLink(string tzFilePath)
             return id;
         }
 
+        /// <summary>
+        /// Enumerate files 
+        /// </summary>
+        private static IEnumerable<string> EnumerateFilesRecursively(string path)
```

Comment:
Could this have been kept in the form it's in FileSystem.Unix.cs, so it's easier to update when that changes?

Reply:
Put another way you kept a subsetted File.cs this could be in a subsetted Directory.cs. I guess it would need bits of FileSystem.Unix.cs etc.

Reply:
The implementation in CoreFX FileSystem.cs is pretty different. It has a lot of extra stuff in it. If this needs to be updated, it is not going to be easy in any case. I would wait and see if the current structure is a problem.